src/main/java/pl/golabm/controller/FamilyController.java

```diff
@@ -37,9 +45,13 @@ public void delete(@PathVariable Long id) {
         for (FamilyMember familyMember : familyService.getById(id).getFamilyMembers()) {
             familyMemberService.delete(familyMember.getId());
         }
+        for (Child child: familyService.getById(id).getChildList()) {
+            childService.delete(child.getId());
+        }
         familyService.delete(id);
     }
 
+    @CrossOrigin(origins = "http://localhost:3000")
```

Comment:
https://stackoverflow.com/questions/44697883/can-you-completely-disable-cors-support-in-spring